docs/DeveloperGuide.adoc

```diff
@@ -753,65 +753,43 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
-|`* *` |user |have a Help command that displays screenshots of positive examples |understand the commands more visually
+|`* *` |debt collector |have profile pictures for my clients |easily identify them in a glance
 
-|`* *` |user |have a user system |allow multiple users to use their own customised addressbook on the same program
+|`* *` |security-concerned debt collector |view my last login time |ensure that no one else is using my account
 
-|`* *` |helpful user |have the ability to send contacts to my friends |ensure that they don't have to key it in from scratch
+|`* *` |security-concerned debt collector |have 2FA authentication for login and every other important action(exporting contacts) |ensure that no one else is capable of accessing sensitive information
 
-|`* *` |user |have 3rd party software(calendar) to draw information from the addressbook |
+|`* *` |multi-device user |be able to export my contacts in another format (e.g xml, vcf) |store my contacts in another device
 
-|`* *` |user |view my last login time |ensure that no one else is using my account
+|`*` |debt collector |create custom fields |store additional information for some contacts
 
-|`* *` |user |destroy my program & backups |wipe the system clean, before passing the program to someone
+|`*` |debt collector |randomly generate a contact from the cleared list(clients who have paid their debts) |choose to chat up a random contact to talk them into getting a loan
 
-|`* *` |user |hide link:#private-contact-detail[private contact details] by default |minimize chance of someone else seeing them by accident
+|`*` |debt collector |use a file upload feature |upload signed documents to a specific client record in the database
 
-|`* *` |user |prevent the addition of duplicate contacts |maintain a unique list of contacts to prevent confusion
-
-|`* *` |user |have profile pictures for my contacts |easily identify them in a glance
-
-|`* *` |user |have a birthday field |receive birthday reminders
-
-|`* *` |user |be able to filter my contacts according to tags |have an overview of who is under which tag
-
-|`* *` |user |be able to export my contacts in another format (e.g xml, vcf) |store my contacts in another device
-
-|`*` |user |be able to send an email to my contacts through the app |send an email without opening another window
-
-|`*` |user with many persons in the address book |sort persons by name |locate a person easily
-
-|`*` |user |add emoticons to contact names |make my addressbook look nicer
-
-|`*` |user |be able to assign short nicknames to contacts |find them faster
-
-|`*` |user |randomly generate a contact |choose to chat up a random contact
-
-|`*` |user |find people by phone number |know whether to pick up a call
-
-|`*` |user |print my contacts on paper in an neat format |pass hard copies to people
+|`*` |debt collector |have a 'add-log' button attached to each client in the database |easily click it whenever I contact the client
```

Comment:
Not obvious what this log is for

Reply:
I think the portion under the 'so that I can' column could be clearer. From our discussions, the benefit is that the user can have a record of when he/she contacted the debtor.

Reply:
Noted. Made the necessary changes.